projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -86,7 +86,12 @@ export interface FeatureConfig {
   /**
    * An optional custom configuration to provide to the generated module.
    */
-  customConfig?: { import: Import[]; content: string };
+  customConfig?: CustomConfig | CustomConfig[];
```

Comment:
To make it simpler let's only allow for arrays

Reply:
I actually like it, as it simplifies usage for cases when we specify only one custom config.
It's also aligned with `addModuleImport()` from `projects/schematics/src/shared/utils/new-module-utils.ts`.

And it doesn't really bring much complexity, IMO, as we just use the `[].concat()` trick.